sdk/include/ndk/rtlfuncs.h

```diff
@@ -4693,6 +4693,46 @@ BOOLEAN
 NTAPI
 RtlGetNtProductType(_Out_ PNT_PRODUCT_TYPE ProductType);
 
+//
+// Synchronization functions
+//
+#if (_WIN32_WINNT >= 0x0600)
```

Comment:
```suggestion
#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
//
// Synchronization functions
//
```
Also usually, in NDK, we would use something like `#if (NTDDI_VERSION >= NTDDI_VISTA)` instead of the WIN32_WINNT.